cherab/core/laser/models/laserspectrum.pyx

```diff
@@ -0,0 +1,56 @@
+from cherab.core.laser.models.laserspectrum_base cimport LaserSpectrum
+from libc.math cimport sqrt, exp, M_PI
+
+cdef class ConstantSpectrum(LaserSpectrum):
+
+    def __init__(self, double min_wavelength, double max_wavelength, int bins):
+
+        super().__init__(min_wavelength, max_wavelength, bins)
+
+    cdef double evaluate(self, double x) except? -1e999:
+        
+        cdef:
+            double spectrum_width
+            int index
+
+        if self._min_wavelength < x < self._max_wavelength:
+            return 1.0/ (self._max_wavelength - self._min_wavelength)
+        else:
+            return 0
+
+cdef class GaussianSpectrum(LaserSpectrum):
+
+    def __init__(self, double min_wavelength, double max_wavelength, int bins, double mu, double sigma):
+
+        self.sigma = sigma
+        self.mu = mu
+        super().__init__(min_wavelength, max_wavelength, bins)
+
+    @property
+    def sigma(self):
```

Comment:
Can you call this `standard_deviation` instead? It would be more readable than just using the symbol typically used in written down equations involving Gaussians.

Reply:
We do use sigma for the neutral beam width at present. I'd prefer it is consistent with that. I do however prefer mean over mu (in your other comment).

options:

1) guassian(mean, sigma)
2) gaussian(mu, sigma)
3) gaussian(mean, stddev)

ie option 1) for now.... though with hindsight 3) may have be better. We can reevaluate this at a later date. Perhaps we can discuss argument standards in the upcoming cherab meeting.

Reply:
Mean I agree with. I would also pick stddev for sigma, because it is more general in describing moments of functions, as @jacklovell pointed out.